__tests__/Unit/Components/Tasks/ProgressText.test.tsx

```diff
@@ -7,20 +7,30 @@ const DefaultProps = {
 };
 
 describe('ProgressText', () => {
-    test('should render save Progress Text', () => {
+    test('should render save Progress Text', async () => {
         const progress = true;
         const { getByText } = render(
-            <HandleProgressText progress={progress} {...DefaultProps} />
+            <>
+                <HandleProgressText
+                    progress={progress}
+                    loading={false}
+                    {...DefaultProps}
+                />
+            </>
         );
-        fireEvent.click(screen.getByText('save Progress'));
-        expect(getByText('save Progress')).toBeInTheDocument();
+        fireEvent.click(screen.getByText('SAVE'));
+        expect(getByText('SAVE')).toBeInTheDocument();
```

Comment:
What are we checking here?

Should the `Save` not disappear after clicking it, and change to `Update` for the same user?